shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -108,6 +108,7 @@
     public static final String STORAGE_CREATE_COUNT = PREFIX + "segmentstore.storage.create_count";      // Counter
     public static final String STORAGE_DELETE_COUNT = PREFIX + "segmentstore.storage.delete_count";      // Counter
     public static final String STORAGE_CONCAT_COUNT = PREFIX + "segmentstore.storage.concat_count";      // Counter
+    public static final String STORAGE_LARGE_CONCAT_COUNT = PREFIX + "segmentstore.storage.small_concat_count"; // Counter
```

Comment:
It says LARGE concat count but with literal value of "small"

Reply:
oh no!

Reply:
Fixed.